docs/src/developing/programming-model/runtime.md

```diff
@@ -113,10 +113,11 @@ Budget](#compute-budget).
 
 With a transaction-wide compute budget the `max_units` cap is applied to the
 entire transaction rather than to each instruction within the transaction. The
-default number of maximum units will calculated per instruction which means the
-sum of the compute units used by each instruction in the transaction must not
-exceed that value. The default number of maximum units allowed matches existing
-values to avoid breaking existing client behavior.
+transaction-wide default maximum number of units will be calculated as the product
+of the instruction count and the existing per-instruction maximum units. This means
```

Comment:
Can you also mention that the default max is clamped at 1.4M compute units?

Reply:
Considered referencing the current constant values, but decided I didn't want to be responsible for keeping them up to date

Reply:
Can we say something like "at the time of writing, this caps at 1.4M but it's subject to change" as a cop out? I think it's useful

Reply:
I see no precedent for citing other constant values in this file. Feel free to add them if you feel strongly!